### PR TITLE
update units in datatable example to its own property

### PR DESCRIPTION
**Pull request by JCoder121**
[Deploy Preview](https://deploy-preview-2690--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
Changes the dataTable examples within the components/DataTable page on the design site. Update the table examples that have columns with units to match the FIgma design and the Grommet dataTable.units property.

#### Where should the reviewer start?
aries-site/src/examples/components/pagination/PaginationTableExample.js
aries-site/src/examples/components/datatable/DataTableExample.js
aries-site/src/examples/components/datatable/DataTableEmptyCellExample.js

#### What testing has been done on this PR?
Visual checks on snapshot to make sure column units match theme on Figma in both light/dark modes. Note: WAVE accessibility plugin (Chrome) indicates low contrast errors for the units in some of the paginationDataTable example only. 

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

#### How should this be manually tested?
Tester's snapshots/site preview dataTable examples should match Figma design.

#### Any background context you want to provide?
Previously, the header property of the dataTable columns contained <Text> for both the column title and the units. Grommet dataTable now directly supports dataTable.units property.

#### What are the relevant issues?

Related to #2441

### CODE REVIEW

**Review by taysea — APPROVED** [review]
Nice work, feels good to simplify and rely on the theme. Think you just need to update yarn.lock one more time @JCoder121

File: aries-site/src/examples/components/datatable/DataTableEmptyCellExample.js [hunk 1]
```diff
@@ -197,14 +197,8 @@ const columns = [
...
     property: 'size',
-    header: (
-      <Text color="text-strong" weight="bold">
-        Size{' '}
-        <Text size="xsmall" weight="normal" color="text">
-          (TiB)
-        </Text>
-      </Text>
-    ),
+    header: 'Size',
```
Comment (britt6612) [inline aries-site/src/examples/components/datatable/DataTableEmptyCellExample.js:200]: nice I didnt even know about this new theme aspect 👍

**Review by britt6612 — APPROVED** [review]
Did you update yarn.lock?

**Comment by taysea** [comment]
> Did you update yarn.lock?

It was actually just out of date w master, so all resolved now.

**Pull request merged**